om-2.0.rdf

```diff
@@ -39440,8 +39726,16 @@
     <om:commonlyHasUnit rdf:resource="&om;IndianRupee"/>
   </owl:Class>
 
-  <owl:Class rdf:about="http://theworldavatar.com/resource/ontouom/AmountOfMoneyPerArea">
-    <om:commonlyHasUnit rdf:resource="http://theworldavatar.com/resource/ontouom/pound_sterling_per_sqm"/>
+  <owl:Class rdf:about="&om;AmountOfMoneyPerArea">
+    <om:commonlyHasUnit rdf:resource="&om;poundSterlingPerSquareMetre"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="&om;AmountOfMoneyPerEnergy">
+    <om:commonlyHasUnit rdf:resource="&om;euroPerMegawattHour"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="&om;AmountOfMoneyPerTimeInterval">
+    <om:commonlyHasUnit rdf:resource="&om;euroPerHour"/>

```

Comment:
One could argue whether those `commonlyHasUnit` relationships are required or the normal `Unit` declarations suffice (but not necessay!)

Reply:
I don't think the ```commonlyHasUnit``` relationships are necessary. My understanding is that they were added to be consistent with OM, where most quantities have both ```commonlyHasUnit``` relationship and the normal ```Unit``` declaration.

Reply:
I missed adding in the definition for ```metreAOD``` and the other water level height units, will add them in.

Reply:
Thanks, what about `litrePerDay`, `litrePerSecond`, and `nanogramPerCubicMetre`?